parser/sql/engine/dialect/doris/src/main/antlr4/imports/doris/DALStatement.g4

```diff
@@ -615,6 +615,15 @@ assignGtidsToAnonymousTransactionsDef
     : (OFF | LOCAL | string_)
     ;
 
+refresh
+    : REFRESH (
```

Comment:
When there are 5 or fewer syntax branches, please write them on one line.

Reply:
I kept them in one line.